lib/ts/recipe/totp/components/themes/translations.ts

```diff
@@ -0,0 +1,36 @@
+import { defaultTranslationsCommon } from "../../../../translation/translations";
+
+export const defaultTranslationsTOTP = {
+    en: {
+        ...defaultTranslationsCommon.en,
+        TOTP_SHOW_SECRET_START: "Unable to scan? Use a",
+        TOTP_SHOW_SECRET_LINK: "secret key",
+        TOTP_SHOW_SECRET_END: "",
+        TOTP_CODE_VERIFICATION_HEADER_TITLE: "Enter TOTP",
+        TOTP_CODE_VERIFICATION_HEADER_SUBTITLE:
+            "Open the two-factor authenticator (TOTP) app on your mobile device to view your authentication code",
+        TOTP_DEVICE_SETUP_HEADER_TITLE: "Enable TOTP",
+        TOTP_DEVICE_SETUP_HEADER_SUBTITLE:
+            "Please scan the given QR code from a phone app like Google Authenticator or Authy.",
```

Comment:
idk if we support authy. I just opened their app, and the TOTP is 7 digits instead of 6. Will the backend and frontend be able to support this somehow?

Reply:
I think we should - but I've removed the reference.

Reply:
Can you confirm this? Please test it out.

Reply:
Tested it, authy does support this (it shows 6 digits, the supertokens logo and the account info pretty nicely)